tests/functional/wdio.conf.js

```diff
@@ -52,7 +52,7 @@ exports.config = {
       "goog:chromeOptions": {
         args: [
           process.env.EQ_RUN_FUNCTIONAL_TESTS_HEADLESS ? "--headless" : "--start-maximized",
-          "--window-size=1280,1090",
+          "--window-size=1280,1100",
```

Comment:
Had to up this again, looks like if an element is outside the immediate window it doesn't work